dll/cpl/desk/lang/pl-PL.rc

```diff
@@ -9,6 +9,19 @@
 
 LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 
+IDD_THEMES DIALOGEX 0, 0, 246, 210
+CAPTION "Themes"
```

Comment:
CAPTION "Kompozycje"
STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
FONT 8, "MS Shell Dlg"
BEGIN
    LTEXT "Kompozycja to Tapeta, zestaw dźwięków, ikon i innych elementów, które pomogą Ci spersonalizować komputer za pomocą jednego kliknięcia.", IDC_STATIC, 5, 5, 235, 30
    LTEXT "Kompozycja:", IDC_STATIC, 5, 42, 55, 10
    COMBOBOX cmb1, 5, 52, 160, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
    PUSHBUTTON "Z&apisz jako...", psh1, 170, 52, 70, 14
    PUSHBUTTON "&Usuń", psh2, 170, 70, 70, 14